scripts/systemd/falcoctl.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Falco: Automatic rules update service
+Documentation=https://falco.org/docs/
+
+[Service]
+Type=simple
+User=root
+ExecStart=/usr/bin/falcoctl artifact follow
+UMask=0077
+TimeoutSec=30
+RestartSec=15s
+Restart=on-failure
+PrivateTmp=true
+NoNewPrivileges=yes
+ProtectSystem=true
+ProtectKernelTunables=true
+RestrictRealtime=true
+#RestrictAddressFamilies=~AF_PACKET
+#StandardOutput=null
```

Comment:
Do we really need `StandardOutput=null` for all other services?
/cc @leogr

Reply:
@FedeDP 
I'm unsure, but we likely need to keep it. The reason is to avoid polluting `/var/logs/messages`. See this my old [comment](https://github.com/falcosecurity/falco/issues/1673#issuecomment-866741461).

Reply:
Ok thank you!